src/components/Toolbar/Toolbar.jsx

```diff
@@ -271,20 +236,27 @@ class Toolbar extends React.Component {
       childrenProps.shape = shape;
     }
 
+    const transitionStyles = {
+      entered: {
+        opacity,
+        transform: 'translate(0,0)'
+      }
+    };
+
     return (
-      <CSSTransitionGroup
-        transitionName="transition"
-        transitionAppear
-        transitionAppearTimeout={100}
-        transitionEnter={false}
-        transitionLeave={false}
-      >
-        <StyledAside className={asideClasses} {...rest}>
-          {React.Children.map(children, child =>
-            React.cloneElement(child, childrenProps)
-          )}
-        </StyledAside>
-      </CSSTransitionGroup>
+      <Transition in appear timeout={100}>
```

Comment:
The Toolbar did not appear with a transition before. I think it's a bit weird to have a transition on that element right after the page is loaded...

Reply:
Right, my bad.